src/cli/api/apihelper.go

```diff
@@ -96,12 +96,12 @@ func (helper *APIHelper) DoRequest(req *http.Request) (*http.Response, error) {
 
 }
 
-func parseErrResponse(raw []byte) (string, error) {
+func parseErrResponse(raw []byte) string {
```

Comment:
This is not correct. you need to take a look at the content type and decide whether to use json unmarshal or not.